CHANGELOG.md

```diff
@@ -1,4 +1,17 @@
-## 3.14.0 (Unreleased)
+## 3.15.0 (Unreleased)
+## 3.14.0 (March 21, 2020)
```

Comment:
You'll want to keep `3.14.0` at the top with the "Unreleased" date, it's a very load-bearing part of the release tooling